### DISCUSSION TITLE
Submenow pack

### DISCUSSION THREAD

**Discussion by Rambooo69** [atalaydenknalbant/autosubscriber-youtube#6] [General]
Why don't you make the submenow script choose the "rocket all in one" pack which gives 40 sub, likes and views instead of the "jet all in one" which gives only 20 of everything..?

**Comment by atalaydenknalbant** [comment]
driver.find_element_by_xpath("//*[@id='buttonPlan8']").click() is button plan 8 last one? I thought I am selecting the right one since I can see my subs accordingly with that selection 
![image](https://user-images.githubusercontent.com/32653649/117465293-35d4fb80-af5a-11eb-975a-0a7f370a16fd.png)

**Reply by Rambooo69** [reply]
Ho sorry.. I didn't realise u changed it.. it used to be 20.. my bad..
